kentikapi/client.go

```diff
@@ -54,10 +68,12 @@ type Config struct {
 	// CloudExportAPIURL defaults to "https://cloudexports.api.kentik.com".
 	CloudExportAPIURL string
 	// SyntheticsAPIURL defaults to "https://synthetics.api.kentik.com".
-	SyntheticsAPIURL string
-	AuthEmail        string
-	AuthToken        string
-	RetryCfg         RetryConfig
+	SyntheticsAPIURL      string
+	SyntheticsGRPCAPIURL  string
+	CloudExportGRPCAPIURL string
+	AuthEmail             string
+	AuthToken             string
+	RetryCfg              RetryConfig
```

Comment:
Please add to RetryConfg docstring the list of services that do not support retries. The same with Timeout and LogPayload.

Reply:
Done